bin-docker/rspec

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-docker-compose run --rm web bundle exec rspec "$@"
+docker-compose run --rm web /bin/bash -c "bundle exec rspec \"$@\"; export tempcode=\$? && rm -Rf tmp/ log/* && exit \$tempcode"
```

Comment:
What is this doing?

Reply:
I do not know how this got committed e.e but it's the hack I'm doing locally because Docker keeps writing tmp/ or log/ files with root permissions.

Let me delete it, whoops.